.github/workflows/workflow.yml

```diff
@@ -14,18 +14,41 @@ jobs:
       matrix:
         os: [ubuntu-latest, macOS-latest, windows-latest]
         node-version: ['10.18.0', '*']
+        # The buildbot pins Netlify Build's dependencies, like `npm ci`.
+        # But other consumers do not, like `npm install`.
+        # So we test both.
+        install-command: ['npm ci', 'npm install']
         exclude:
+          # `npm install` is only run in latest Node.js on Ubuntu
```

Comment:
GitHub actions matrix include/exclude are very verbose:
  - `include` does not work as an allow list (which I don't find very intuitive). Instead it only adds back combinations removed by `exclude`.
  - `exclude` requires specifying each matrix dimension value.